scanners/subfinder/parser/parser.js

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: the secureCodeBox authors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+async function parse(fileContent) {
+  if (!fileContent)
+    return [];
+
+  const targets = parseResultFile(fileContent);
+  return transformToFindings(targets);
+}
+
+function transformToFindings(targets) {
+  return targets.map((item) => ({
+      "name": item.host,
+      "identified_at": null,
+      "description": `Found subdomain ${item.host}`,
+      "category": "Subdomain",
+      "location": item.host,
+      "osi_layer": "NETWORK",
+      "severity": "INFORMATIONAL",
+      "attributes": {
+        "addresses": {
+          "ip": item?.ip || null
+        },
+        "domain": item.input,
+        "hostname": item.host,
+        "ip_address": item?.ip || null,
```

Comment:
we've updated (most, i think) scanners a while back to have an attributes.ip_addresses field with an array of ips. (as a host can resolve to multiple ones)

would be good to have the same here too.